dev/TabView/TabView.xaml

```diff
@@ -683,23 +709,87 @@
 
                             <Button x:Name="CloseButton"
                                 Grid.Column="2"
-                                Width="{ThemeResource TabViewItemHeaderCloseButtonSize}"
-                                Height="{ThemeResource TabViewItemHeaderCloseButtonSize}"
-                                FontSize="{ThemeResource TabViewItemHeaderCloseFontSize}"
                                 Margin="{ThemeResource TabViewItemHeaderCloseMargin}"
                                 Content="&#xE711;"
                                 IsTextScaleFactorEnabled="False"
                                 IsTabStop="False"
-                                Background="{ThemeResource TabViewItemHeaderCloseButtonBackground}"
-                                Foreground="{ThemeResource TabViewItemHeaderCloseButtonForeground}"
-                                Style="{StaticResource TabViewCloseButtonStyle}"
+                                Style="{ThemeResource TabViewCloseButtonStyle}"
                                 HighContrastAdjustment="None" />
                         </Grid>
                     </Grid>
-
                 </ControlTemplate>
             </Setter.Value>
         </Setter>
     </Style>
 
+    <Style x:Name="TabViewScrollButtonStyle" TargetType="RepeatButton">
+        <Setter Property="Background" Value="{ThemeResource TabViewButtonBackground}"/>
+        <Setter Property="Foreground" Value="{ThemeResource TabViewButtonForeground}"/>
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}"/>
+        <Setter Property="FontSize" Value="{ThemeResource TabViewItemScrollButonFontSize}"/>
+        <Setter Property="FontFamily" Value="{ThemeResource SymbolThemeFontFamily}"/>
+        <Setter Property="Width" Value="{ThemeResource TabViewItemScrollButtonWidth}"/>
+        <Setter Property="Height" Value="{ThemeResource TabViewItemScrollButtonHeight}"/>
+        <Setter Property="FontFamily" Value="{ThemeResource SymbolThemeFontFamily}"/>
+        <Setter Property="FocusVisualMargin" Value="-3"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="RepeatButton">
+                    <ContentPresenter x:Name="ContentPresenter"
+                        Background="{TemplateBinding Background}"
+                        contract7Present:BackgroundSizing="{TemplateBinding BackgroundSizing}"
+                        Content="{TemplateBinding Content}"
+                        ContentTemplate="{TemplateBinding ContentTemplate}"
+                        ContentTransitions="{TemplateBinding ContentTransitions}"
+                        contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
+                        contract7NotPresent:CornerRadius="{ThemeResource TabViewItemButtonCornerRadius}"
+                        FontSize="{TemplateBinding FontSize}"
```

Comment:
These font properties are propogating, so setting them explicitly shouldn't be necissary.